### PR TITLE
Commits folders file path.

### PR DESCRIPTION
**Pull request by ghost**
Completion 1/5
Understanding 1/5

Fails to import seed file. I believe I must have messed up some part of the set up. I tried to add what I could in to seeds.rb without being able to test it.

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+Apartment.create(address: "10707 Weymouth Street", monthly_rent: 1000 ,sqft: 5000,num_beds: 3, num_baths: 4)
```
Comment (jshawl) [inline db/seeds.rb:1]: if you require your models:

```
require_relative "../models/apartment"
```

your seeds should work!

File: models/apartment.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class Apartment < ActiveRecord::Base
+  has_many :tenants
+  attr_accessor :address, :monthly_rent, :sqft, :num_beds, :num_baths, :renters
+
+  def initialize
```
Comment (jshawl) [inline models/apartment.rb:5]: remove initialize method so active record can handle object creation

**Comment by jshawl** [comment]
:+1: Let me know if my comment doesn't fix the seeds issue - would be happy to walk through this with you.

**Comment by ghost** [comment]
Thanks for the feedback Jesse. I played around in there a lot tonight and still can't get it working. Your direction certainly helped. I didn't realize I was supposed to be doing  '''ruby seeds.rb''' I thought I was "importing" in using the psql landlord_db < seeds.rb mechanism like schema. 

That being said, now that I've made your changes and am running seeds directly I am now getting the following issue, all my seeds are importing with nil data but are not creating errors.

`1] pry(main)> Tenant
=> Tenant(id: integer, name: string, age: integer, gender: string, apartment_id: integer)
[2] pry(main)> Tenant.create(name: "Davis", age: 30, gender: "Male", apartment_id: 1)
=> #<Tenant:0x007fbf8b889530 id: 21, name: nil, age: nil, gender: nil, apartment_id: 1>
[3] pry(main)> Tenant.create(name: "Davis", age: 30, gender: "Male", apartment_id: 1)
=> #<Tenant:0x007fbf8b8495e8 id: 22, name: nil, age: nil, gender: nil, apartment_id: 1>
[4] pry(main)> Tenant.new(name:"Davis")
=> #<Tenant:0x007fbf89b44e48 id: nil, name: nil, age: nil, gender: nil, apartment_id: nil>
[5] pry(main)> Tenant.new(name:'Davis')
=> #<Tenant:0x007fbf8a2a3658 id: nil, name: nil, age: nil, gender: nil, apartment_id: nil>
[6] pry(main)> Tenant.all
=> [#<Tenant:0x007fbf8d140998 id: 1, name: nil, age: nil, gender: nil, apartment_id: 1>,
 #<Tenant:0x007fbf8d140858 id: 2, name: nil, age: nil, gender: nil, apartment_id: 2>,
 #<Tenant:0x007fbf8d140718 id: 3, name: nil, age: nil, gender: nil, apartment_id: 2>,
 #<Tenant:0x007fbf8d1405d8 id: 4, name: nil, age: nil, gender: nil, apartment_id: 2>,
 #<Tenant:0x007fbf8d140498 id: 5, name: nil, age: nil, gender: nil, apartment_id: 1>,
 #<Tenant:0x007fbf8d140358 id: 6, name: nil, age: nil, gender: nil, apartment_id: 3>,
 #<Tenant:0x007fbf8d140218 id: 7, name: nil, age: nil, gender: nil, apartment_id: 3>,
 #<Tenant:0x007fbf8d1400d8 id: 8, name: nil, age: nil, gender: nil, apartment_id: 1>,
 #<Tenant:0x007fbf89adfe80 id: 9, name: nil, age: nil, gender: nil, apartment_id: 1>,
 #<Tenant:0x007fbf89adfc00 id: 10, name: nil, age: nil, gender: nil, apartment_id: 1>,
 #<Tenant:0x007fbf89adfa20 id: 11, name: nil, age: nil, gender: nil, apartment_id: 1>,
 #<Tenant:0x007fbf89adf4a8 id: 12, name: nil, age: nil, gender: nil, apartment_id: 2>,
 #<Tenant:0x007fbf89adec60 id: 13, name: nil, age: nil, gender: nil, apartment_id: 2>,
 #<Tenant:0x007fbf89ade8c8 id: 14, name: nil, age: nil, gender: nil, apartment_id: 2>,
 #<Tenant:0x007fbf89ade1e8 id: 15, name: nil, age: nil, gender: nil, apartment_id: 1>,
 #<Tenant:0x007fbf89addfb8 id: 16, name: nil, age: nil, gender: nil, apartment_id: 3>,
 #<Tenant:0x007fbf89adde50 id: 17, name: nil, age: nil, gender: nil, apartment_id: 3>,
 #<Tenant:0x007fbf89adda68 id: 18, name: nil, age: nil, gender: nil, apartment_id: 1>,
 #<Tenant:0x007fbf89add860 id: 19, name: nil, age: nil, gender: nil, apartment_id: 1>,
 #<Tenant:0x007fbf89add658 id: 20, name: nil, age: nil, gender: nil, apartment_id: 1>,
 #<Tenant:0x007fbf89add450 id: 21, name: nil, age: nil, gender: nil, apartment_id: 1>,
 #<Tenant:0x007fbf89add1a8 id: 22, name: nil, age: nil, gender: nil, apartment_id: 1>]
[7] pry(main)>`

I'm leaning towards just starting over, I feel like I must have done something wrong. I thought I followed along with the tunr thing well enough. Would it be possible to grab some one-on-one time tomorrow and get me back on track?

**Pull request closed without merging**